Makefile

```diff
@@ -348,11 +349,11 @@ endif
 .PHONY: build-windows
 build-windows:
 	@echo "===> Building Antrea bins and antrea/antrea-windows Docker image <==="
-	docker build --cache-from antrea/base-windows:$(WIN_BUILD_TAG) -t antrea/base-windows:$(WIN_BUILD_TAG) -f build/images/base-windows/Dockerfile $(WIN_BUILD_ARGS) .
+	docker build --cache-from antrea/base-windows:$(WIN_BUILD_TAG) -t antrea/base-windows:$(WIN_BUILD_TAG) -f build/images/base-windows/Dockerfile --network $(DOCKER_NETWORK) $(WIN_BUILD_ARGS) .
```

Comment:
Does it mean the user must specify parameter `$(DOCKER_NETWORK)`, otherwise the command would be "docker build .... --network default"? I think the expected behavior should be, check if `$(DOCKER_NETWORK)` is set firstly, and only append parameter "--network $(DOCKER_NETWORK)" when it is set?

Reply:
The docker manual showed "--network string          Set the networking mode for the RUN instructions during build (default "default")" , which means the parameter is default if the user don't specify the parameter. So we can keep "--network" here.

Reply:
OK, thanks for the explanation